src/Components/Link/index.js

```diff
@@ -2,9 +2,17 @@ import React, { Component } from 'react';
 import './index.css'
 
 export default class ApplyLink extends Component {
-    render() {
-        return (
-            <a className="ApplyLink" href={this.props.LinkUrl} > <span className="ApplyLink-text">{this.props.children}</span> </a>
-        )
-    }
-} 
+  render() {
+    return (
+      <a className={this.props.classname} href={this.props.LinkUrl}>
+        {' '}
```

Comment:
What is the purpose of this? Why do we need an empty object? (Just trying to understand & going for a learning moment here! 😬)